.github/workflows/pssanalyzer.yml

```diff
@@ -13,5 +13,5 @@ jobs:
   analyze:
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+      - uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3
```

Comment:
Could you revert this? I think `https://github.com/actions/checkout/commit/8e5e7e5ab8b370d6c329ec480221332ada57f0ab` is the latest